cmd/swapcli/main.go

```diff
@@ -667,10 +667,12 @@ func runGetOngoingSwap(ctx *cli.Context) error {
 		fmt.Printf("Receiving: %s %s\n", info.ExpectedAmount.Text('f'), receivedCoin)
 		fmt.Printf("Exchange Rate: %s ETH/XMR\n", info.ExchangeRate)
 		fmt.Printf("Status: %s\n", info.Status)
+		fmt.Printf("Time status was last updated: %s\n", info.LastStatusUpdateTime.Format(common.TimeFmtSecs))
 		if info.Timeout0 != nil && info.Timeout1 != nil {
 			fmt.Printf("First timeout: %s\n", info.Timeout0.Format(common.TimeFmtSecs))
 			fmt.Printf("Second timeout: %s\n", info.Timeout1.Format(common.TimeFmtSecs))
 		}
+		fmt.Printf("Estimated time to completion: %s\n", info.EstimatedTimeToCompletion)
```

Comment:
The output seems a little overly exact:
```
Estimated time to completion: 20m45.316266949s
```
I didn't test this, but you might be able to use the duration `Round` function passing it 1 minute or one second in the place that you are creating it.

Reply:
good point lol, I rounded it to the nearest second.